### PR TITLE
Add new `bazelrc` field

### PR DESCRIPTION
**Pull request by colinhoglund**
This change adds a `bazelrc` plugin field for specifying an override `.bazelrc`.

The first commit simply refactors some plugin methods to simplify testing, so it will be easier to review per-commit.

Here is some output for a successful local exec using the example repo:

```
...
[publish-one:44] 2020/04/07 19:18:14 Successfully pushed Docker image to ********/drone-bazelisk-ecr-example-one:one
...
[publish-two:37] 2020/04/07 19:19:40 Successfully pushed Docker image to ********/drone-bazelisk-ecr-example-two:two
```

### CODE REVIEW

File: cmd/drone-bazelisk-ecr/plugin.go [hunk 1]
```diff
@@ -43,12 +48,32 @@ func newPlugin() (plugin, error) {
...
 
-	return p, nil
+	return nil
+}
+
+func (p *plugin) getArgs() []string {
+	var args []string
+
+	// append startup options
+	if p.Bazelrc != "" {
```
Comment (dskatz) [inline cmd/drone-bazelisk-ecr/plugin.go:58]: Should we test if this file exists?  Or does bazel fail if this argument points to a file that does not exist?  
If the behavior of bazel is to ignore the argument if it does not exist, we should throw an error.

Comment (colinhoglund) [reply cmd/drone-bazelisk-ecr/plugin.go:58]: good point, i'll look into this.

Comment (dskatz) [reply cmd/drone-bazelisk-ecr/plugin.go:58]: I'll ok with this being added on iteratively.

**Review by dskatz — APPROVED** [review]
Lgtm

File: cmd/drone-bazelisk-ecr/plugin.go (hunk 1 above)
Comment (colinhoglund) [reply cmd/drone-bazelisk-ecr/plugin.go:58]: bazelisk errors out, so leaving as is for now.

```
[ERROR 17:56:49.418 src/main/cpp/option_processor.cc:375] Error: Unable to read .bazelrc file '/drone/src/.bazelrc.drone'.
```

**Pull request merged**